src/components/sustainability/FeatureSectionV2.tsx

```diff
@@ -0,0 +1,50 @@
+import { SustainabilityText } from '@/assets/types/sustainabilty';
+
+export interface FeatureSectionV2 {
```

Comment:
Use the interface. Also, use Props at the end (`FeatureSectionV2Props`)

Reply:
Done